### ISSUE TITLE
some commands missing

### ISSUE THREAD

**Issue by ElitaJaun** [flass/pantagruel#9]
Hi Florent, 

I am getting these two errors as the scripts seems to struggle to locate the following commands:
```
environ_pantagruel_databasename.sh: line 139: interproscan: command not found
```
```
pantagruel_pipeline/pantagruel/scripts/pipeline/pantagruel_pipeline_06_gene_trees.sh: line 120: convalign: command not found
```
Bw, 
E

**Comment by flass** [comment]
Hi Elita, 

thank you for reporting again!
the dependency installation script was missing Python package bioscripts.convert and and the Interproscan version check was not robust to random file being put in the Interproscan FTP... all should be fixed with e71d49e and 9ad2f97.
Please run the install_dependencies.sh script again.

Cheers
Florent

**Comment by ElitaJaun** [comment]
Great! however, now there is a new bug....

```
This is Pantagruel pipeline version 9ad2f970294b068fc99a17b9687c08862c506110 using source code from repository '/home/elita/pantagruel_pipeline/pantagruel'
# will run tasks: 4
[2019-03-28 17:48:20] Pantagrel pipeline step 4: use InterProScan to functionally annotate proteins in the database.
/home/elita
Error: the installed verison of InterProScan (found at /home/elita/pantagruel_pipeline/interproscan-5.34-73.0/interproscan.sh) is 5.34-73.0, different from the version currently operated by the EBI: data.
Using this outdated version would cause the look-up service not to work and thus a significant loss of efficiency.
Please install the most recent version by running again the script '/home/elita/pantagruel_pipeline/pantagruel/install_dependencies.sh'.
Exiting now.
```

**Comment by flass** [comment]
Ah that was the same error as in the install script that needed the same fix; done in 7f59d18.
